_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Creating dashboards
+nav_order: 30
+has_children: true
+---
+
+# Creating dashboards
+
+A dashboard combines multiple data visualizations into a single view. With the **Dashboard** application, you can:
+
+- Get started with pre-built panels and dashboards and customize display properties to create the dashboard you need.
```

Comment:
It seems like this list should either describe the general capabilities of dashboards (what you can do, which would more closely map to the sections of the tutorial, or describe the utility of dashboards (why it's useful). This first item seems like neither. Maybe something along the lines of

```suggestion
- Create a collection of visualizations that can be filtered and customized
```